test/workers/scrape.test.ts

```diff
@@ -15,15 +15,25 @@ import { SearchResultStatus } from '@/constants/enum';
 describe('Scrape:', () => {
   let mockJob: Partial<Job>;
   let userPayload: User;
+  let searchResultPayload: SearchResult;
+  const keyword = 'node';
+  const uuid = 'a4bfbcb1-b9c5-4476-99f0-20756f4974bd';
+  const userId = 3;
 
   beforeAll(() => {
-    userPayload = { ...user, id: 1 };
+    userPayload = { ...user, id: 3 };
```

Comment:
Shouldn't this use the constant defined above?

```suggestion
    userPayload = { ...user, id: userId };
```

This kind of complexity is why it is best not to use before block for creating fixtures. Prefer to move all of this logic to each test. There is come code repetitions but it is far easier to understand and maintain.

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/64/commits/6cc410ac2b91328f0e02761b613ac7e84945ddf1